tasks/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM --platform=linux/amd64 python:3.8-bullseye
 
 RUN apt-get update &&\
-    apt-get install -y gdal-bin libgdal-dev wget unzip python3-rtree python3-all-dev tesseract-ocr ghostscript
```

Comment:
tessereact/ghostcript not required now that NOS_OCR runs the OCR/text extraction instead.